### PR TITLE
Update Karmada docs directory

### LINKED ISSUE #116: Karmada docs directory tidy
**What would you like to be added**:
Now Karmada's docs looks a little messy and it's unfriendly to users. We need to tidy the Karmada docs directory.

**Why is this needed**:
Tidy Karmada's docs directory. I recommend the below:
.
├── Core Concepts 
│   ├── Resource template, pp, cpp, op, cop
├── Key Features 
├── Get Started //A small example for new users.
├── Tutorials //Guide for users to learn how to use key feature step by step.
├── Installation 
├── User Guide //Mainly used to introduce Karmada and its ecological features.
├── Administrator Guide //For administrator, including upgrade, migration, operation and maintenance.
├── Developer Guide //For developer, including secondary development, debug, configuration based on Karmada.
├── Contributor Guide //For contributor, including how to contribute to Karmada community.
├── Case Study //end-user cases.
├── Troubleshooting
└── FAQ
└── Reference

Organize existing docs：https://github.com/Poor12/karmada/tree/master/website_docs

TODO-list
- [ ] `Core Concepts`  Replenish
- [ ] `Key Features`  Replenish
- [ ] `Get Started` Replenish
- [ ] `User Guide`  FederatedHPA Replenish
- [ ] `User Guide` 全局资源视图聚合API部分与统一认证鉴权需分离
- [ ] `User Guide` 全局资源视图补充统一运维（exec、log）部分
- [ ] `User Guide` 全局资源视图补充资源、事件全局搜索部分
- [ ] `User Guide` 角色关注分离统一认证鉴权需与聚合API部分分离
- [ ] `User Guide` 角色关注分离补充全局资源配额部分
- [ ] `User Guide` 角色关注分离补充可复用调度策略部分
- [ ] `Administrator Guide` 补充迁移部分（包含单集群迁移、从kubefed迁移）
- [ ] `Administrator Guide` 补充Karmadactl指导手册
- [ ] `Administrator Guide` 补充monitoring监控日志部分
- [ ] `Developer Guide` 补充如何调试karmada部分
- [ ] `Developer Guide` 补充对Karmada-scheduler做自定义配置部分
- [ ] `Reference` Replenish

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind cleanup
/kind documentation

**What this PR does / why we need it**:
Update Karmada docs directory. Refer to #116 

**Which issue(s) this PR fixes**:
Fixes #116 

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/get-started/nginx-example.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
```
Comment (RainbowMango) [inline docs/get-started/nginx-example.md:1]: Where is the tutorial directory?

File: docs/administrator/karmadactl/bash-auto-completion-on-linux.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
-# bash auto-completion on Linux
+---
+title: Bash auto-completion on Linux
```
Comment (RainbowMango) [inline docs/administrator/karmadactl/bash-auto-completion-on-linux.md:2]: This seems the user guide or developer guide, not the administrator guide.

**Comment by Poor12** [comment]
All suggestions are done.

**Pull request merged**